plugins/adr/package.json

```diff
@@ -45,6 +45,7 @@
     "react-router-dom": "6.0.0-beta.0 || ^6.3.0"
   },
   "devDependencies": {
+    "@backstage/catalog-client": "workspace:^",
```

Comment:
Is this dep being used somewhere ? I couldn't find it being imported in this plugin

Reply:
@kuangp I have removed the @backstage/catalog-client dependency. I have also removed the octokit and git-url-parse dependencies as they were left over from the old implementation and not being used anymore.